packages/components/src/text/Text.js

```diff
@@ -0,0 +1,50 @@
+// @flow
+
+import * as React from 'react';
+import { Platform } from 'react-native';
+import {
+  Text as UniversalComponentsText,
+  StyleSheet,
+} from '@kiwicom/universal-components';
+
+/**
+ * This wrapper is necessary because loading fonts with Expo does not allow using fontWeight and fontStyle
+ * cf. https://docs.expo.io/versions/latest/guides/using-custom-fonts/
+ */
+
+export default function Text(
+  props: $PropertyType<typeof UniversalComponentsText, 'props'>,
+) {
+  if (Platform.OS !== 'ios' && Platform.OS !== 'android') {
```

Comment:
Will this ever be true?

Reply:
Oh, I read that wrong, but `Platform.OS == 'web'` maybe?

Reply:
I think in that case, it needs to be the case specifically for Android and iOS since these are our target environments with Expo. Your suggestion would work too, but I guess my proposal is a bit safer for the future? I don't feel so strongly about it, so I can change it if you prefer.

Reply:
No, let's keep it.